packages/api-rpc/src/api/request_tokens.ts

```diff
@@ -0,0 +1,66 @@
+import { Api, JsonRpc } from 'eosjs';
+import JsSignatureProvider from 'eosjs/dist/eosjs-jssig';
```

Comment:
We've just merged a branch to `develop` which updates `eosjs` version to `20.0.0`. So `JsSignatureProvider` is now being exported as a constant so you'll need to do `import { JsSignatureProvider } from 'eosjs/dist/eosjs-jssig';`.

Reply:
JsSignatureProvider import has been updated :-)